external/CMakeLists.txt

```diff
@@ -208,7 +214,11 @@ if(USE_DISCORD_RICH_PRESENCE)
 		${CMAKE_BINARY_DIR}/external/discord_game_sdk/cpp/voice_manager.cpp)
 
 	if(APPLE)
-		target_link_libraries(discord-rpc PUBLIC "${CMAKE_BINARY_DIR}/external/discord_game_sdk/lib/x86_64/discord_game_sdk.dylib")
+		if(ARCHITECTURE STREQUAL "arm64")
+			target_link_libraries(discord-rpc PUBLIC "${CMAKE_BINARY_DIR}/external/discord_game_sdk/lib/x86_64/discord_game_sdk.dylib")
```

Comment:
I realize this is obsolete and will probably never get merged anyways, but it looks like x86_64 and aarch64 are reversed here

Reply:
I already aware of it and gonna make PR for arm(aarch64) macOS after macdu PRs aarch64 things

Reply:
I'll try to make a PR with my aarch64 modifications next week, sorry for the delay.